java/Anton-Kozyk/src/com/zoolatech/lecture6/tasks/_3/UserService.java

```diff
@@ -18,7 +18,8 @@ public String findUserEmail(String id) {
         try {
             return userRepository.findUserEmail(id);
         } catch (UserMissingException e) {
-            return e.getLocalizedMessage();
+            System.out.println(e.getLocalizedMessage());
         }
+        return "Something went wrong...";
```

Comment:
it's better to return null: the expected output of the method is an email, and not this phrase